lib/cli/lib/detect.js

```diff
@@ -23,6 +23,13 @@ function detectFramework(dependencies) {
     return types.VUE;
   }
 
+  if (
```

Comment:
odd that order matters...

Reply:
@gabrielcsapo I had to add react to my apps package.json as a temporary work around and from that point it was determined to be a react app. In switching them around, it checks for ember first, react second and correctly determines the type. 

Figured it was an easy fix to something that could catch others out.